es-core/src/GuiComponent.h

```diff
@@ -45,7 +45,7 @@ class GuiComponent
     void setSize(const Eigen::Vector2f& size);
     void setSize(float w, float h);
     virtual void onSizeChanged() {};
-	
+
```

Comment:
You can revert this file (checkout from another revision), and commit, then squash - there are tutorials on git rebasing etc around the web.